ls1intum/Artemis
# Enhancement/guided tour/disable tour for mobile

### Checklist

### Description
The guided tour is currently not optimized for mobile devices. Therefore I disabled the `Start Tour` button for mobile phones and tablets and added the `//TODO` remark for a future task.

### Steps for Testing
1. Log in to Artemis
2. Click on account-menu
3. Start tour from account menu
4. Redo the steps from a mobile phone or tablet
5. The `Start tour` menu item should not be visible

### Screenshots
![IMG_9856](https://user-images.githubusercontent.com/33764166/64612398-fac0a480-d3d3-11e9-9291-216d55a8bb85.PNG)
